### PR TITLE
History Phalcon

### PR DESCRIPTION
**Pull request by duythien**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,6 +124,10 @@ Need:
 - Joomla! Framework 1: <http://www.joomla.org/announcements/release-news/5521-joomla-framework-1-0-released.html>
 
 - Slim 2.3.0: <https://github.com/codeguy/Slim/releases/tag/2.3.0>
+- Phalcon 1.1 <http://blog.phalconphp.com/post/49778100441/phalcon-1-1-released>
+
+## 2014
+- Phalcon 1.2.5 <http://blog.phalconphp.com/post/73222509376/phalcon-1-2-5-released>
```
Comment (sergeyklay) [inline README.md:130]: The 1.2.5 is still 1.x

Comment (duythien) [inline README.md:130]: @sergeyklay 

I should remove 1.2.5 or 1.1 ?

Comment (sergeyklay) [inline README.md:130]: as I understand it, needs only 1.0, 2.0, 3.0 etc. but no 1.1.1, 1.0.2, 2.1.0

Comment (duythien) [inline README.md:130]: yes. I see commit of you about Phalcon, maybe I will close my PR

**Pull request closed without merging**